### PR TITLE
Costumer Create made

### PR DESCRIPTION
**Pull request by asqqer**

### CODE REVIEW

File: src/main/java/com/outofbounds/empire/Costumers/Controllers/Costumerscontroller.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@RestController
+public class Costumerscontroller {
+
+
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Costumers/Controllers/Costumerscontroller.java]: Remove spacing

File: src/main/java/com/outofbounds/empire/Costumers/Controllers/Costumerscontroller.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    public @ResponseBody
+    CostumerModel costumerModel(@PathVariable String phoneNumber) {return repositorieCostumer.findByPhoneNumber(phoneNumber);}
+
+    /*@RequestMapping(method = RequestMethod.GET, value = "/movies/generate/random")*/
+}
+
+
+
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Costumers/Controllers/Costumerscontroller.java]: Remove spacing

File: src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
+package com.outofbounds.empire.Costumers.Models;
+import javax.persistence.*;
+
+@Entity
+@Table(name = "Costumers")
+public class CostumerModel {
+
+
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java]: Spacing

File: src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    public class Movie {
+
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java]: Spacing

File: src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        public int getId() {
+            return id;
+        }
+
+        public void setId(int id) {
+            this.id = id;
+        }
+
+        public String getPhonumber() {
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java]: Typo getPhoneNumber

File: src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+
+    public class Movie {
+
+        @Id
+        @GeneratedValue
+        @Column(name = "id", unique = true, nullable = false)
+        private int id;
+
+        @Column(name = "Phonenumber")
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java]: Typo

File: src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+        public void setId(int id) {
+            this.id = id;
+        }
+
+        public String getPhonumber() {
+            return phonumber;
+        }
+
+        public void setPhonumber(String phonumber) {
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java]: Typo

File: src/main/java/com/outofbounds/empire/Costumers/Repostitories/RepositorieCostumer.java [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Transactional
+@Repository
+public interface RepositorieCostumer extends CrudRepository<CostumerModel, Integer> {
+
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Costumers/Repostitories/RepositorieCostumer.java]: Spacing

File: src/main/java/com/outofbounds/empire/Costumers/Repostitories/RepositorieCostumer.java [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+        public List<CostumerModel> findAll();
+        public CostumerModel findById(int Id);
+        public CostumerModel findByPhoneNumber(String Phonenumber);
+
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Costumers/Repostitories/RepositorieCostumer.java]: Spacing

File: src/main/java/com/outofbounds/empire/Costumers/Repostitories/RepositorieCostumer.java [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+    }
+
+
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Costumers/Repostitories/RepositorieCostumer.java]: Spacing

File: src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+        public int getEmail() {
+            return Email;
+        }
+
+        public void setEmail(int email) {
+            Email = email;
+        }
+    }
+
+
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java]: Spacing

File: src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+        @Id
+        @GeneratedValue
+        @Column(name = "id", unique = true, nullable = false)
+        private int id;
+
+        @Column(name = "Phonenumber")
+        private String phonumber;
+
+        @Column(name = "Name", nullable = true)
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java]: Names should start with small

File: src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+        private String name;
+
+        @Column(name = "Email")
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java]: email

File: src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java (hunk 12 above)
Comment (jasonkellydk) [reply src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java]: name

File: src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java (hunk 6 above)
Comment (jasonkellydk) [reply src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java]: phone_number

File: src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java [hunk 14]
```diff
@@ -0,0 +1,65 @@
+package com.outofbounds.empire.Costumers.Models;
+import javax.persistence.*;
+
+@Entity
+@Table(name = "Costumers")
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Costumers/Models/CostumerModel.java]: Typo customers

File: src/main/java/com/outofbounds/empire/Costumers/Controllers/Costumerscontroller.java [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+     * Basic route for movies
+     *
+     * @return Movie
+     */
+    @CrossOrigin(origins = "http://localhost:8000")
+    @RequestMapping(method = RequestMethod.GET, value = "/movies")
+    public @ResponseBody
+    List<CostumerModel> movies() {
+        return repositorieCostumer.findAll();
+    }
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Costumers/Controllers/Costumerscontroller.java]: Spacing

File: src/main/java/com/outofbounds/empire/Costumers/Controllers/Costumerscontroller.java [hunk 16]
```diff
@@ -0,0 +1,43 @@
+package com.outofbounds.empire.Costumers.Controllers;
+
+import com.outofbounds.empire.Costumers.Repostitories.RepositorieCostumer;
+import com.outofbounds.empire.Costumers.Models.CostumerModel;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Costumers/Controllers/Costumerscontroller.java]: Spacing

**Review by jasonkellydk — COMMENTED** [review]
Hey checkout these fixes

File: src/main/java/com/outofbounds/empire/Costumers/Repostitories/RepositorieCostumer.java (hunk 10 above)
Comment (asqqer) [reply src/main/java/com/outofbounds/empire/Costumers/Repostitories/RepositorieCostumer.java]: done

**Review by AnderskdKnudsen — APPROVED** [review]
well done, hombre

**Pull request merged**